src/component/variantPage/BasicInfo.tsx

```diff
@@ -319,6 +321,30 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
         );
     }
 
+    private jsonButton() {
+        return (
+            <DefaultTooltip
+                placement="top"
+                overlay={
+                    <span>Click to read more about the raw API response</span>
+                }
+            >
+                <Link
+                    to={
+                        '/annotation/' +
+                        this.props.variant +
+                        '?fields=hotspots%2Cmutation_assessor%2Cmy_variant_info%2Cptms%2Cannotation_summary'
```

Comment:
I include hotspots, mutation assessor, my variant info, ptm, annotation summary. Any other fields I need to add?

Reply:
that looks good to me 👍